internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -265,25 +253,23 @@ func convertEFSPermissions(input map[string]manifest.Volume) ([]*template.EFSPer
 		if volume.EFS.UseManagedFS() {
 			continue
 		}
+		// We're not using a managed volume and the EFS config is empty.
```

Comment:
Should we move this comment to 3 lines above and say something more descriptive, such as  "We only need to add additional permissions in the case of BYO EFS system".